### PR TITLE
test: Vspecinteg

### PR DESCRIPTION
**Pull request by BasilBeirouti**
*Issue #, if available:*

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I certify that the changes I am introducing will be backword compatible, and I have discussed concerns about this, if any, with the Python SDK team
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [ ] I have passed the region in to all S3 and STS clients that I've initialized as part of this change.
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added unit and/or integration tests as appropriate to ensure backward compatibility of the changes
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [ ] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: tests/integ/test_marketplace.py [hunk 1]
```diff
@@ -186,6 +197,134 @@ def predict_wrapper(endpoint, session):
...
+        NameContains="my-flower-detection-model",
+        SortBy="CreationTime",
+        SortOrder="Descending",
+    )
+
+    if len(response["ModelPackageSummaryList"]) > 0:
+        sm_client.delete_model_package(ModelPackageName=model_name)
+
+    # assert that response is non-empty
+    assert len(response["ModelPackageSummaryList"]) > 0
```
Comment (mufaddal-rohawala) [inline tests/integ/test_marketplace.py:325]: Can we somehow check if the `ValidationSpecification` is part of the model response?

Comment (BasilBeirouti) [reply tests/integ/test_marketplace.py:325]: yes we can use describe-model function to return the JSON description of the model. Then we can assert that 
 verifyformarketplace=True.

Comment (mufaddal-rohawala) [reply tests/integ/test_marketplace.py:325]: I think ^ this didn't work out right? Can you add details/reasoning and close this conversation.

**Comment by mufaddal-rohawala** [comment]
@BasilBeirouti can you update the details/overview section with details and also change PR title per [guidelines](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#commit-your-change)

**Pull request closed without merging**